format/src/main/thrift/carbondata.thrift

```diff
@@ -130,6 +130,7 @@ struct DataChunk2{
     9: optional list<binary> encoder_meta; // Extra information required by encoders
     10: optional BlockletMinMaxIndex min_max; 
     11: optional i32 numberOfRowsInpage;
+    12: optional list<i32> bitmap_encoded_dictionaries;
```

Comment:
store each chunk' dictionaries, for example if in 2 chunks one, have dictionary 2 and 3, and the other have 3 and 4. There are different between the 2 chunks. So bitmap_encoded_dictionaries to store the dictionaries for each chunk.